app/main.py

```diff
@@ -1,2 +1,117 @@
 class Dictionary:
-    pass
+
+    def __init__(self):
+        self.capacity = 8
+        self.threshold = int(self.capacity * (2 / 3))
+        self.length = 0
+        self.base_data = [None for i in range(self.capacity)]
+
+    def __setitem__(self, key, value):
+        a = 0
```

Comment:
unnecessary logic

Reply:
But the code without this logic does not work

Reply:
you should create a logic without this variable

Reply:
you should fix this case